src/framework/preferences/messages/enhanceInputs_en_US.json

```diff
@@ -0,0 +1,6 @@
+{
+    "label": "Enhance Inputs",
+    "description": "Emphasize links, buttons, menus, textfields, and other inputs",
+    "switchOn": "on",
+    "switchOff": "off"
```

Comment:
There seems to be some inconsistency for adjusters that have a the on/off switch. In some message bundles the values ore ON/OFF and others on/off. Could you please update them to be consistent. The rendered casing is probably handled by CSS, but it's worthwhile to double check.